src/python/pants/backend/jvm/tasks/coverage/jacoco.py

```diff
@@ -60,6 +61,16 @@ def create(self, settings, targets, execute_java_for_targets):
                                                    scope='jacoco')
       return Jacoco(settings, agent_path, cli_path, targets, execute_java_for_targets)
 
+  # TODO(jtrobec): deprecate these options and move them to subsystem scope
```

Comment:
Given that these are brand new options, you should be able to skip straight to defining them on this subsystem with shorter names.

For example, if you move this option to the `def register_options` method above, you should be able to just name it `--target-filters`, which would allow you to specify it with `--jacoco-target-filters` (and see it in the help for `./pants help jacoco`).